src/core/event/index.js

```diff
@@ -34,15 +34,82 @@ export function Events(Base) {
     }
 
     initEvent() {
+      const { coverpage, keyBindings } = this.config;
+      const keyModifiers = ['alt', 'ctrl', 'meta', 'shift'];
+
       // Bind toggle button
       this.#btn('button.sidebar-toggle', this.router);
       this.#collapse('.sidebar', this.router);
       // Bind sticky effect
-      if (this.config.coverpage) {
+      if (coverpage) {
         !isMobile && on('scroll', this.__sticky);
       } else {
         body.classList.add('sticky');
       }
+      // Bind keyboard shortcuts
+      if (keyBindings && keyBindings.constructor === Object) {
+        // Prepare key binding configurations
+        Object.values(keyBindings || []).forEach(bindingConfig => {
+          const { bindings } = bindingConfig;
+
+          if (!bindings) {
+            return;
+          }
+
+          // Convert bindings to arrays
+          // Ex: 'alt+t' => ['alt+t']
+          bindingConfig.bindings = Array.isArray(bindings)
+            ? bindings
+            : [bindings];
+
+          // Convert key sequences to arrays
+          // Ex: ['alt+t', 'ctrl+t'] => [['alt', 't'], ['ctrl', 't']]
+          bindingConfig.bindings = bindingConfig.bindings.map(keys => {
+            if (typeof keys === 'string') {
+              keys = keys.split('+');
+            }
+
+            return Array.isArray(keys)
+              ? keys.map(k => k.toLowerCase().trim())
```

Comment:
Shall we sorted the keys here? otherwise, such as the `meta+i` and `i+meta` gonna be the different keyBind , but I think they should be the same.

Reply:
> Shall we need pre check and sort the keys? Such as the meta+i and i+meta gonna be the different keyBind currently, although it will both be called , I think they should be the same.

What kind of pre-checking are you thinking of?

Sorting isn't necessary (key order doesn't matter), but it's not a bad idea to normalize the key bindings. I've updated the PR so that key combinations are now normalized by forcing all to lowercase and sorting alphabetically while ensuring modifiers come before non-modifier keys. For example:

- `alt+a` => `['alt', 'a']` (no change)
- `a+alt` => `['alt', 'a']` (modifier first)
- `a+shift+meta+ctrl+alt` => `['alt', 'ctrl', 'meta', 'shift', 'a']` (modifiers first & alpha sort)

> About the case sensitive conditions, such as I and i, we treat them are the same key now.
Should they be the same key or not?

The are case insensitive. The key bindings for lowercase and uppercase letters are different:

- Lowercase "i" = `"i"` or `"I"` 
- Uppercase "I" = `"shift+i"`, `"shift+I"`, `"Shift+I"`, `"SHIFT+I"`, etc.